tests/integration/config/test_provider_env.py

```
@@ -0,0 +1,493 @@
+
+        text_cfg, vision_cfg = _get_llama_cpp_configs()
+
+        assert text_cfg.extra_params["n_gpu_layers"] == 32
+        assert vision_cfg.extra_params["n_gpu_layers"] == 32
+
+    def test_invalid_n_gpu_layers_is_ignored(self, monkeypatch: pytest.MonkeyPatch) -> None:
+        monkeypatch.setenv("FO_LLAMA_CPP_MODEL_PATH", "/models/llama3.gguf")
+        monkeypatch.setenv("FO_LLAMA_CPP_N_GPU_LAYERS", "not_a_number")
+
+        with patch("file_organizer.config.provider_env.logger.warning") as mock_warn:
+            text_cfg, vision_cfg = _get_llama_cpp_configs()
```

Ruff (0.15.9)

[warning] Unpacked variable `vision_cfg` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)